deploy/deploy-systemd.sh

```diff
@@ -0,0 +1,64 @@
+#!/bin/bash
+
+set -e
+
+export LC_ALL=C.UTF-8
+export LANG=C.UTF-8
+
+BRANCH=$1
+name=
+
+if [ "x$BRANCH" = "x" ]; then
+  echo "Please pass branch to deploy as first argument"
+  exit 1
+fi
+
+if [ "x$BRANCH" = "xproduction" ]; then
+  name=foodsaving-world
+else
+  name=foodsaving-world-dev
+fi
+
+function restart-workers() {
+  sudo systemctl restart "${name}-worker.target"
+}
+
+backend_dir="/var/www/${name}/www"
```

Comment:
Hm yes. Well if we go for celery in the foreseeable future, it would need another systemd service. Or this recommended python alternative: http://supervisord.org
Which could also make sense to use for daphne and the workers.